dmgroups.tex

```diff
@@ -0,0 +1,105 @@
+
+\subsection{DM System Engineering Team \label{sect:sysengt}}
+The System engineering team is led by the DM Project Manager and looks
+after all aspects of system engineering. It is comprised of not only a System Engineer (\secref{role:sysengineer}) but also the Requirements Engineer (\secref{role:reqeng}), Software Architect (\secref{role:softarc}), Operations Architect (\secref{role:opsarc}) and the Pipeline Scientist (\secref{role:pipe}).
+
+While the product owners help DM to create the correct product , fit for purpose, the DM System engineering team must ensure we do it correctly. This group concerns its self with DM (sub)system wide decisions on architecture and software engineering.
+
+Within this  group we must:
+\begin{itemize}
+\item Formalise the Product list/tree for DM, these are not the data products but the DM software and systems which produce the products.
+\item Formalise the documentation tree for DM - which documents need to be produced for each product.
+\item Agree how to trace the baseline requirements verification and validation status.
+\item  \ldots
+\end{itemize}
+ Some of these tasks are obviously delegated to the individuals in the group. These individuals also are the conduit to/from the rest of the DM team to raise ideas/issues with the engineering approach.
+
+ \subsubsection{Communications}
+ The DM System engineering team will only physically meet to discuss specific topics; there will not be a regular meeting of the group outside of the one to one meetings with the DM project manager for the individuals in the group.
+Discussions will be held via email until in person talks are required.
+
+\subsection{DM Leadership Team} \label{sect:dmlt}
+
+The DM Leadership Team (DMLT) purpose is to establish scope of work and resource allocation across DM and ensure overall project management integrity across DM.
+The following mandate established the DMLT:
+
+\begin{itemize}
+\item Charter/purpose
+	\begin{itemize}
+	\item Maintain scope of work and keep within resource allocation across DM
+	\item Ensure overall project management integrity across DM
+	\item Ensure Earned Value management requirements are met
+	\end{itemize}
+\item Membership
+	\begin{itemize}
+	\item Co-Chaired by the DM Project Manager  and  DM Project Scientist
+	\item Lead Institution Technical/Control Account Managers (T/CAMs or CAMs)
+	\item Institutional Scientists
+	\item Members of the DM Systems Engineering Team (\secref{sect:sysengt})
+	\end{itemize}
+\item Responsibilities
+	\begin{itemize}
+	\item Prepares all budgets, schedules, plans
+	\item Meets every week to track progress, address issues/risks, adjust work assignments and schedules, and disseminate/discuss general PM communications
+	\item Creates and publishes monthly, quarterly, annual progress reports
+	\item Meets at start of each software development phase with SAT to establish detailed scope/work plan
```

Comment:
I don't think we meet with the Science Archives Team at ESAC... Is SAT the (no longer existing) System Architecture Team?